drivers/sensor/maxim/max31855/max31855.c

```diff
@@ -68,7 +68,7 @@ static int max31855_channel_get(const struct device *dev, enum sensor_channel ch
 
 	switch (chan) {
 
-	case SENSOR_CHAN_AMBIENT_TEMP:
+	case SENSOR_CHAN_THRMCP_TEMP:
```

Comment:
This change make sense to it match with sensor feature, but just a note it will break existing user code.

Reply:
Yes, I am certainly aware of it breaking user code. I tried it also in a different way, which doesn't break user code, following the implementation of max31855 in mcp9600, look here: https://github.com/zephyrproject-rtos/zephyr/pull/70349

But nothing happened there and the only comment was that I should do it the proper way with a sensor-specific channel :-)

Can you see an alternative way that doesn't break user code?